crawler/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "crawler"
+version = "0.1.0"
+authors = ["Simon Bernier St-Pierre <git@sbstp.ca>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
This looks like a left over?

Reply:
This is created by `cargo new`

Reply:
But should we leave it here? I usually remove it first thing after creating a new crate.